api_docs/construct-narrow.md

```diff
@@ -51,7 +51,11 @@ important optimization when fetching messages in certain cases (e.g.
 when [adding the `read` flag to a user's personal
 messages](/api/update-message-flags-for-narrow)).
 
-**Changes**: In Zulip 9.0 (feature level 250), support was added for
+**Changes**: In Zulip 9.0 (feature level 262), support was added for a new
+filter, `with`, which returns messages in the same channel and topic as that
+of the operand of the filter, with the first unread message as anchor.
```

Comment:
You should also document the `with` parameter semantics in the section on `### Message IDs`, below.

Reply:
updated!